NEWS.md

```diff
@@ -387,6 +387,22 @@
 
 41. `tables()` is faster by default by excluding the size of character strings in R's global cache (which may be shared) and excluding the size of list column items (which also may be shared). `mb=` now accepts any function which accepts a `data.table` and returns a higher and better estimate of its size in bytes, albeit more slowly; e.g. `mb = utils::object.size`.
 
+42. `setnames` with numeric `old` will now repeat `new` to the same length as `old` to match the behavior of `names(x)[old]<-new` including for negative indices, [#5853](https://github.com/Rdatatable/data.table/issues/5853). Thanks to @michaelchirico for reporting and @TimothyWillard for implementing.
```

Comment:
This should be moved up to the new features of `1.15.99`

Reply:
Ah, I thought only the latest was in `NEWS.md`, didn't see the version headings, thanks!